src/Operator/Operators.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Operator;
+
+use Twig\Operator\Binary\AbstractBinaryOperator;
+use Twig\Operator\Unary\AbstractUnaryOperator;
+
+class Operators implements \IteratorAggregate
+{
+    private array $operators = [];
```

Comment:
I suggest adding types for those properties (so that the static analysis job running in our CI can analyze the code of this class more precisely, bring more value): `@var array<value-of<OperatorArity>, array<string, OperatorInterface>>`